pkg/lib/authn/identity/info.go

```diff
@@ -182,6 +199,8 @@ func (i *Info) DisplayID() string {
 		return i.Biometric.KeyID
 	case model.IdentityTypePasskey:
 		return i.Passkey.CreationOptions.PublicKey.User.DisplayName
+	case model.IdentityTypeSIWE:
+		return i.SIWE.Address
```

Comment:
Is there any convention to format chain ID along with address?

Reply:
Seems that EIP681's target address can also be a single user instead of a contract as well.

https://eips.ethereum.org/EIPS/eip-681